--- ../../../ParaView-3.4.0.orig/Qt/Components/CMakeLists.txt	2008-09-04 04:07:18.000000000 +0900
+++ CMakeLists.txt	2008-11-08 09:52:36.463000000 +0900
@@ -147,6 +147,7 @@
   pqOutputPortComboBox.h
 #  pqParticleTracerPanel.h
   pqPQLookupTableManager.h
+  pqPOpenFOAMPanel.h
   pqPipelineBrowser.h
   pqPipelineBrowserContextMenu.h
   pqPipelineBrowserStateManager.h
@@ -536,6 +537,8 @@
   pqOptionsPage.h
   pqOutputPortComboBox.cxx
   pqOutputPortComboBox.h
+  pqPOpenFOAMPanel.cxx
+  pqPOpenFOAMPanel.h
   pqPQLookupTableManager.cxx
   pqPQLookupTableManager.h
 #  pqParticleTracerPanel.cxx
