--- ../../../ParaView-3.4.0.orig/Qt/Components/pqObjectInspectorWidget.cxx	2008-09-23 04:15:14.000000000 +0900
+++ pqObjectInspectorWidget.cxx	2008-11-08 09:57:23.455000000 +0900
@@ -62,6 +62,7 @@
 #include "pqLoadedFormObjectPanel.h"
 #include "pqObjectBuilder.h"
 #include "pqObjectPanelInterface.h"
+#include "pqPOpenFOAMPanel.h"
 #include "pqParticleTracerPanel.h"
 #include "pqPipelineSource.h"
 #include "pqPluginManager.h"
@@ -150,6 +151,10 @@
         {
         return new pqExodusIIPanel(proxy, p);
         }
+      if(QString("OpenFOAMReader") == proxy->getProxy()->GetXMLName())
+        {
+        return new pqPOpenFOAMPanel(proxy, p);
+        }
       if(QString("XdmfReader") == proxy->getProxy()->GetXMLName())
         {
         return new pqXDMFPanel(proxy, p);
@@ -190,6 +195,7 @@
       if(QString("ExodusReader") == proxy->getProxy()->GetXMLName() ||
          QString("ExodusIIReader") == proxy->getProxy()->GetXMLName() ||
          QString("ExodusRestartReader") == proxy->getProxy()->GetXMLName() ||
+         QString("OpenFOAMReader") == proxy->getProxy()->GetXMLName() ||
 #ifdef PARAVIEW_USE_SILO
          QString("SiloReader") == proxy->getProxy()->GetXMLName() ||
 #endif
