--- CMakeLists.txt.orig	2009-05-23 00:33:16.000000000 +0900
+++ CMakeLists.txt	2009-07-23 18:05:00.000000000 +0900
@@ -155,6 +155,7 @@
   pqOutputPortComboBox.h
 #  pqParticleTracerPanel.h
   pqPQLookupTableManager.h
+  pqPOpenFOAMPanel.h
   pqPipelineBrowser.h
   pqPipelineBrowserContextMenu.h
   pqPipelineBrowserStateManager.h
@@ -576,6 +577,8 @@
   pqOrbitWidget.h
   pqOutputPortComboBox.cxx
   pqOutputPortComboBox.h
+  pqPOpenFOAMPanel.cxx
+  pqPOpenFOAMPanel.h
   pqPQLookupTableManager.cxx
   pqPQLookupTableManager.h
 #  pqParticleTracerPanel.cxx
