--- pqObjectInspectorWidget.cxx.orig	2009-05-23 00:33:17.000000000 +0900
+++ pqObjectInspectorWidget.cxx	2009-07-23 18:05:12.000000000 +0900
@@ -60,6 +60,7 @@
 #include "pqLoadedFormObjectPanel.h"
 #include "pqObjectBuilder.h"
 #include "pqObjectPanelInterface.h"
+#include "pqPOpenFOAMPanel.h"
 #include "pqParticleTracerPanel.h"
 #include "pqPipelineSource.h"
 #include "pqPluginManager.h"
@@ -139,6 +140,10 @@
         {
         return new pqExodusIIPanel(proxy, p);
         }
+      if(QString("OpenFOAMReader") == proxy->getProxy()->GetXMLName())
+        {
+        return new pqPOpenFOAMPanel(proxy, p);
+        }
       if(QString("XdmfReader") == proxy->getProxy()->GetXMLName())
         {
         return new pqXDMFPanel(proxy, p);
@@ -178,6 +183,7 @@
       {
       if (QString("ExodusIIReader") == proxy->getProxy()->GetXMLName() ||
          QString("ExodusRestartReader") == proxy->getProxy()->GetXMLName() ||
+         QString("OpenFOAMReader") == proxy->getProxy()->GetXMLName() ||
 #ifdef PARAVIEW_USE_SILO
          QString("SiloReader") == proxy->getProxy()->GetXMLName() ||
 #endif
