--- CMakeLists.txt.orig	2009-03-18 03:45:22.000000000 +0900
+++ CMakeLists.txt	2009-08-29 21:16:15.000000000 +0900
@@ -79,6 +79,7 @@
 vtkMoleculeReaderBase.cxx
 vtkNetCDFCOARDSReader.cxx
 vtkNetCDFReader.cxx
+vtkNewOpenFOAMReader.cxx
 vtkOBJReader.cxx
 vtkOutputStream.cxx
 vtkOpenFOAMReader.cxx
@@ -344,6 +345,14 @@
 ENDIF ( VTK_USE_ODBC AND ODBC_FOUND )
 
 #-----------------------------------------------------------------------------
+# Regex
+# See if a POSIX Regex library exists on this system. Used for
+# vtkNewOpenFOAMReader.
+#
+INCLUDE(${VTK_CMAKE_DIR}/FindRegex.cmake)
+
+
+#-----------------------------------------------------------------------------
 # Add these classes only if vtkTypeUInt64Array exists
 IF(VTK_HAS_UINT64_ARRAY)
   SET(Kit_SRCS
